### PR TITLE
Removes earthsblood from the game, adds a new Ambrosia branch, changes around ambrosia's chems.

### PR DESCRIPTION
**Pull request by CRITAWAKETS**
##  About The Pull Request

This PR removes earthsblood from the entire game, along with replacing it with psicodine in amborisa gaia, a chemical that was not previously available to botany that has uses.

## Why It's Good For The Game

Earthsblood has always been ridiculously overpowered. In the case of golden trays, it just permitted botany to completely ignore doing its own job. In the case of as a chemical, the healing effect has always obsoleted every single other healing chemical in the game even after the nerf. It was like giving adminoriazine to botany. It completely removed any sense of realism and interdepartmental cooperation, as botany could just use its magic chemical for itself. It does not belong in the current medical system and the current botany system. In addition, player and coder feedback seems to be in favor of nerfing botany massively.

TL:DR, this is an absolutely overpowered chemical that completely breaks the balance of the game. Every botanist who uses this is powergaming but has to because its so blatantly overpowered.

## Changelog
:cl:
del: Removed earthsblood from the game.
add: Added ragnarokia, a plant that can mutate from ambrosia deus. It contains all the stuff you do not want.
tweak: The chemicals in ambrosia plants have been shuffled around for a tiered progression without earthsblood.
/:cl:

### CODE REVIEW

**Comment by TheVekter** [comment]
For context, what does Piscodine do?

**Comment by CRITAWAKETS** [comment]
Supresses bad mood and turns it into natural mood. Also reduces jitteriness, dizziness, confusion and disgust. Basically, put it in the food to make people feel good.

**Comment by TheVekter** [comment]
Okay, what benefit does it have for plants? Or are we just completely removing the concept of getting something that makes it easier to do their job?

**Comment by carlarctg** [comment]
i'd like it if self sustaining trays were still a thing so you can put say holy melons in the brig without needing to mantain them

**Comment by TheVekter** [comment]
Yeah, I don't like this. I feel like the end game for Botany is to eventually not have to worry about weeds and bugs and just grow whatever you want. Removing this seems silly instead of just removing the healing factor and making it harder to get.

**Comment by kriskog** [comment]
You guys realize earthsblood was nerfed right? It is barely strong anymore, unless you give it almost a minute to work and accept being a pacifist until its gone.
#46258

**Comment by EdgeLordExe** [comment]
i really really dislike this change. this will make end game botany neigh impossible as you will need to split your attention 20 fold. like i understand where you are coming from but golden trays are something you have to do if you even want to leave your department for a second.

**Comment by Mickyan** [comment]
this is stupid psicodine is an artificial chemical, change earthsblood to have a similar effect if you have to

**Comment by Dorsisdwarf** [comment]
>Am I glad he's frozen in there and that we're out here, and that he's the sheriff and that we're frozen >out here, and that we're in there, and I just remembered, we're out here. What I wanna know is: >Where's the caveman?

I cant believe I missed the skooks reference

**Comment by Shadowflame909** [comment]
>Removing Earthsblood for "good-moodlet" drug

We already have weed in the game Critawaket.

Gaia making you pacifist while it's in your system in exchange for healing you is a good nerf.

Charcoal doesn't exist anymore so you cant just flush out chemicals without cost. (Unless you want to use Multiliver, and get some lung damage. But that counters the point of healing.) Only mitigating toxins otherwise.

Tldr; Adminorazadine doesn't come with a downside. So your false equivalent isn't really helping your case if you don't have an in-game reason for why it still needs a nerf. You can't power-game Earthsblood anymore.

**Comment by Couls** [comment]
psicodon't do this

**Comment by HomocidlePancakes** [comment]
Ah yes, very good change. It was so fun to manually water and feed plants, especially when you had 20 of them! That feel of empowerment that comes when your job is so taxing, that you can't leave your department while doing it, I hope more people will end up experiencing it and understand how great it is!

**Comment by Ratvar** [comment]
I like the way botanists call 5-15 minute plant their "endgame", really shows how broken their progression is

**Comment by HomocidlePancakes** [comment]
>I like the way botanists call 5-15 minute plant their "endgame", really shows how broken their progression is.
You know, maybe having there is something wrong with the job being so tedious that you have to rush self-sustaining trays for it to be playable?

**Comment by ATH1909** [comment]
> along with replacing it with psicodine in amborisa gaia, a chemical that was not previously available to botany that has uses.

```
/obj/item/seeds/cannabis/rainbow
	name = "pack of rainbow weed seeds"
	desc = "These seeds grow into rainbow weed. Groovy... and also highly addictive."
	icon_state = "seed-megacannabis"
	species = "megacannabis"
	plantname = "Rainbow Weed"
	product = /obj/item/reagent_containers/food/snacks/grown/cannabis/rainbow
	mutatelist = list()
	reagents_add = list(/datum/reagent/colorful_reagent = 0.05, /datum/reagent/medicine/psicodine = 0.03, /datum/reagent/drug/happiness = 0.1, /datum/reagent/toxin/mindbreaker = 0.1, /datum/reagent/toxin/lipolicide = 0.15)
	rarity = 40
```

https://github.com/tgstation/tgstation/blob/master/code/modules/hydroponics/grown/cannabis.dm

In case you can't read code, what I'm trying to say here is that botany _already_ has access to psicodine.

Also, even though ambrosia gaia/earthsblood is a trap plant/chem for newer botanists, it still shouldn't be just flat-out removed from the game. It gives people brain damage and makes them pacifistic, which I think is enough of a downside, IMO. In short, we already nerfed ambrosia gaia, and a chem that you usually need to devote 15 minutes or so to making in order to produce it in reasonable quantities _should_ be much more effective than chems that you can make in less than 1 minute with a chem dispenser.

File: code/modules/hydroponics/hydroponics.dm [hunk 1]
```diff
@@ -27,7 +27,7 @@
 	var/unwrenchable = 1
 	var/recent_bee_visit = FALSE //Have we been visited by a bee recently, so bees dont overpollinate one plant
 	var/using_irrigation = FALSE //If the tray is connected to other trays via irrigation hoses
-	var/self_sufficiency_req = 20 //Required total dose to make a self-sufficient hydro tray. 1:1 with earthsblood.
+	var/self_sufficiency_req = 20 //Required total dose to make a self-sufficient hydro tray. 1:1 with adminoriazine.
```
Comment (ATH1909) [inline code/modules/hydroponics/hydroponics.dm]: You do know that adminordrazine already has an effect when used on plant trays, right?

Also, it's spelled "adminordrazine", not "adminoriazine".

**Comment by Farquaar** [comment]
As someone with over 50 hours clocked as botanist and many more as chef, let me say this:

If you want to divert more attention to gene-modding and less to farming, try being useful to the station so science upgrades your tray efficiency. Try cultivating your plants so they don't die after being unattended for 3 minutes. If you use mutagen, consider looking at stats other than just potency and yield. Personally, I never use gaia and I can easily maintain 8+ trays while attending to other projects and roleplay interactions.

While a discussion can be had as to whether outright removal is the way to go, eternally starvation-proof, weed-proof, pest-proof trays need a hefty nerf if we want to balance botany properly.

**Comment by OverlyGeneric** [comment]
Unnecessary as the previous earthsblood's nerf has made it unviable for combat usage as stated by numerous people in this friend.

I enjoy you as a friend, CRITAWAKETs, but this PR is redundant for removing earthsblood for it's healing properties.

Although Farquaar does makes a good point with his comment:

> While a discussion can be had as to whether outright removal is the way to go, eternally starvation-proof, weed-proof, pest-proof trays need a hefty nerf if we want to balance botany properly.

If you want botany to cooperate more, you should start by removing properties that make self-sustaining trays too good like their weed and pest protection.

You could also force botanists to maintain their self-sustaining trays with some negative consequences for laziness.

You could do something simple like immortal cockroaches or you could do something silly like having tiny primitive humanoids crawl out of the tray to attack people with toothpick-sized spears because of **!!deeplore!!** evolution juice in ambrosia gaia.

**Comment by CatDerpington** [comment]
If anything botany needs nerfing. Botanists have it easy with Gaia. It’s basically rush to get Gaia. Convert trays so they can do things like make space drugs or exploding lemons. 

I rarely see a botanist actually use trays to make food for the chef. It’s always rushing to do meme gimmicks. Botany needs a makeover in tune of what cobby did to medical.

**Comment by kriskog** [comment]
> 
> 
> Botany needs a makeover in tune of what cobby did to medical.

Botany needs a lot more of a makeover than that. Slaving over weeding and watering trays instead of working on mutations and having fun is bad gameplay. We need to make there be less maintenance to get rid of the need for gaia trays, and make a more interesting mutation/splicing system than mutagen and the DNA machine. Right now anyone can make super deadly shit in 15 minutes if they try, and experimentation is pointless since you can always make the exact same thing again and again. My deathfruit recipe havent changed in in the last year, and its not likely to either.

**Comment by CRITAWAKETS** [comment]
Maybe if you botanists actually did your job then maybe you would realize this isnt such a bad change.

I can play a game of "Guess who's powergaming botany" by looking at all the people who think that earthsblood is nessesary for plant trays. Its only nessesary if you just take botanist and greytide. Plants do not die quickly if you take care of them especially if the trays are upgraded.

Also, mannitol + earthsblood basically completely invalidates cobbymed and is the second best medicine in the game if you dont OD on it. Sure, if you pacifism, but if you control the dose then you dont even get the pacifism effect. And with the pacifism the healing ramps up to ludicrous levels.

File: code/modules/hydroponics/hydroponics.dm (hunk 1 above)
Comment (CRITAWAKETS) [reply code/modules/hydroponics/hydroponics.dm]: Yes, but its effect doesnt really add any self-sufficiency to the plants. Its more for when the admins want to make a meme plant tray.

**Comment by kriskog** [comment]
> 
> 
>Sure, if you pacifism, but if you control the dose then you dont even get the pacifism effect. 

Thats just plain wrong. Pacifism is active from the first chem tick, and you need 25 chem ticks until the healing ramps up. You are 50 seconds as a pacifist before it ramps, and then for however long you wish to have good healing.

**Comment by CRITAWAKETS** [comment]
Changed the chemical from psicodine (which was infact already obtainable) to granibitaluri, which wasnt and is going to be useful in cobbychem going forwards.

**Comment by CRITAWAKETS** [comment]
> Thats just plain wrong. Pacifism is active from the first chem tick, and you need 25 chem ticks until the healing ramps up. You are spending 50 seconds as a pacifist before it ramps, and then for however long you wish to have good healing.

Ah, right.

Its still completely and utterly ludicrous in its healing compared to most of everything.

**Comment by necromanceranne** [comment]
CRIT, while get what you're trying to do, making the chem gaia produces (if it's to be granibitaluri) be worse than the stuff out of Ambrosia and Deus before it is a little baffling.

I'd recommend something a bit more in line with being three mutations deep and suggest making it potentially split into one of two different potential outcomes for the third mutation to prevent consistently being able to get Gaia, much like how the various cannabis strains work and making it hard to get the SPECIFIC one you want.

Then shuffle the healing chems up through the mutations. Gaia then makes omnizine (which is still a stupidly good chem and makes acquiring it en masse just that bit harder as gaia has to be babysit constantly), make it an INHERENT gene (we have that functionality now) so you can't produce it via taking it out of gaia and putting it into wheat or something, and creating an alternative mutation from Gaia that is probably less desirable for healing but still potentially worth it for other reasons (aka make an evil gaia). 

Baseline Ambrosia could probably just have a lot of vitamins which makes it super healthy (which sticks with it's theme of being a kind of super medicinal herb and foodstuff) and desirable for medicinal purposes in other ways. Just get rid of the toxin, that probably shouldn't be in there in the first place.

Deus being able to provide actual medicine makes it the most ideal medicinal plant and also means you can still get healing plants in botany without having to worry about babysitting the plant too much, though with the catch of it being one with side effects.

This would probably solve more of the issues without completely gutting the hell out gaia with a hatchet. The only issue remains is what would be the alternative plant.

**Comment by CRITAWAKETS** [comment]
Alright, here's the plan then. Im going to take Anne's plan for this.

Vulgaris has cobbychem, like quite literally every "basic" C2 in it.
Deus has grani.
Gaia has Omnizine
Evil Ambrosia (Demonia?) has the Toxin.

**Comment by cacogen** [comment]
>I like the way botanists call 5-15 minute plant their "endgame", really shows how broken their progression is.

no that's the startgame, then you get to start growing the shit you actually want (well, the first version of it anyway)

**Comment by CRITAWAKETS** [comment]
Ragnarokia has been added as an "Evil" version of ambrosia now. It has all the bad stuff with some more like phlogiston and chlorine trifluoride.
Deus now has grannybutaloni.
Gaia now has omnizine.
Vulgaris doesnt have drugz anymore.

**Comment by ExcessiveUseOfCobblestone** [comment]
gaia is indeed a crutch when stock parts fill the niche of LESS (but not complete maintenance-free farms).

I just nerfed it tho from a healing aspect so full on remove seems to be a tad overkill imo.

**Comment by Shadowflame909** [comment]
Another issue is healing chems keep getting targetted for being "over-powered" and "game-breaking" when their necessity is simply just a reaction to the special super-duper station genociding antag weapons.

Remember the e-sword and e-bow.

It's not overpowered, it's actually quite balanced.

It's like complaining how easy it is to revive from death, and ignoring how easy it is to die.

**Comment by carlarctg** [comment]
i can't believe you actually thought granibitaluri was a good chem to put in ambrosia gaia, you really do have no idea what you're doing or are just being malicious to botanists

**Comment by dootdoom** [comment]
>Nerf botany at all
>I CANT BELIEVE YOU'D NERF BOTANY IT'S FAIR AND BALANCED
>glow mushrooms
>plant guns
>earths blood
>holy melons
yes please merge

**Comment by Dawson1917** [comment]
👍 Because botany's absolute requirement to grow gaia every round to fix the painful finickiness of plant trays is bad. A real fix is needed.

**Comment by TheVekter** [comment]
> 👍 Because botany's absolute requirement to grow gaia every round to fix the painful finickiness of plant trays is bad. A real fix is needed.

Here's a question - why the fuck would you ever remove something like this and not propose a fix for the actual problem, which is Botany being frustrating and boring to play without earthsblood? You know what's going to happen if this gets merged? People just stop playing Botany wholesale. That's it.

@CRITAWAKETS Close this shit until someone can come up with a better alternative.

**Comment by Shadowflame909** [comment]
> > Nerf botany at all
> > I CANT BELIEVE YOU'D NERF BOTANY IT'S FAIR AND BALANCED
> > glow mushrooms
> > plant guns
> > earths blood
> > holy melons
> > yes please merge

Outside of plant guns, which is only obtained by specific xenobio cooperation. Where's the unbalance?

**Comment by ATH1909** [comment]
> Ragnarokia has been added as an "Evil" version of ambrosia now. It has all the bad stuff with some more like phlogiston and chlorine trifluoride.
> Deus now has grannybutaloni.
> Gaia now has omnizine.
> Vulgaris doesnt have drugz anymore.

Wow. I didn't think that you could make this PR even worse, but you somehow did.

The current big three problems with botany that people complain about are:
1. Tending to plants before getting to the fun gene modding is tedious; it's the botany equivalent of mining's ash storms.
2. The RNG in getting plant mutations from your mutagen can be very frustrating/annoying (this is admittedly a lesser problem compared to the other two, but still indeed a problem).
3. People don't like the strength of the weaponized plants that botany can pump out (I'm personally fine with them, but I'm not gonna argue against that point here).

Having seen these three problems, you decided to go with a ""solution"" that makes each of them _worse_.
1. Self-explanatory. The benefit gained from golden trays is currently an appropriate reward to the investment needed to grow reasonable quantities of earthsblood. Some botanists really don't like the "thrilling gameplay" of tending to your plants that is "click plant with water when water low, click plant with robust harvest when nutrients low, click plant with weed-removing stick when weeds high, click plant with pest spray when pests high", and ambrosia gaia/earthsblood provides a way around that for them. Do you think that anyone is going to enjoy botany more because you made it more tedious?
2. You really, genuinely think that making getting the wrong plant when your plant DOES happen to mutate is going to cause anything other than RNG-induced frustration and rage? I guarantee you that someone is going to try to mutate their ambrosia deus into ambrosia gaia 5 times and get ragnarokia all 5 times, and that is going to absolutely ruin their round/day. NOBODY is going to like whiffing that 50-50 shot, and its only purpose will be to frustrate people and cause random delays in their progress that are mostly out of their control.
3. Hmm, yes, people are complaining about the kill chems that botany can stuff into their plants. I know, I'll add a new plant that has _even more_ kill chems in it! Seriously, what the fuck do you expect people to use ragnarokia for, other than setting people on fire AND poisoning them with the same plant? There are two outcomes here: One, ragnarokia's chems are good enough to be used in optimized plant deathmixes, in which case you've made the problem worse, or two, they're not good enough to be used in optimized plant deathmixes, in which case you've created a mostly useless plant whose only purpose is to say "fuck you, you lost a 50/50 coin flip, better hope you were growing a second plant" to people who want to grow ambrosia gaia?

Of course, nobody's going to want to grow ambrosia gaia anymore (except for the people who haven't played in a while or who are just blindly following the wiki), because you've made it suck ASS now. News flash for you, buddy, omnizine is an AWFUL chem. Say you chug 15u of omnizine. Congratu-fucking-lations, you're healing at a rate of 0.25 points of each damage type per second (remember, a chem tick occurs every 2 seconds, not every second), which I shouldn't have to tell you is slow as SHIT (even outside of a combat situation). "still a stupidly good chem" my ass.

Alright, that devolved into a bit of a swear-fest at the end there, but my point is: Why are you trying to "balance" botany by making it more tedious? Why are you trying to nerf a chem that's already been nerfed to being in a good place? How often do you actually play botanist? Cobby, why the hell did you assign this PR to yourself instead of closing this abomination the moment you read it (and its comments)? You're honestly scaring me here, Cobby.

**Comment by ATH1909** [comment]
**Grod's Law: You cannot and should not balance bad mechanics by making them annoying to use.**

**Comment by FancyCyclops** [comment]
Don't bother talking.

**Comment by ATH1909** [comment]
Note: I've been informed by necromanceranne that the reason that they said that omnizine was a "good chem" was because that you could team up with xenobio to mass-produce regen jelly with an omnizine-producing plant.

... But we already HAVE an omnzine-producing plant (two, in fact): ambrosia deus, which is currently less difficult to obtain than how difficult ambrosia gaia would be to obtain if this PR gets merged, and I have yet to see a single botanist team up with xenobio to mass-produce regen jelly yet. In short, if people aren't combo'ing omnizine from botany and slime jelly from xenobio now (when it's easier to do so), they almost certainly still won't do that if/after this PR gets merged. Also, gating the major benefits of botany's hardest to obtain/grow plants behind cooperation with xenobio sounds like a very bad idea.

**Comment by ATH1909** [comment]
> Don't bother talking.

Hm? Are you saying that no one here can be reasoned with, or do you just not like long essays on why I think that certain changes would be bad ideas to make?

**Comment by TheVekter** [comment]
This PR is a thinly veiled attempt to remove something that one of his targets probably used in a traitor round to stay alive so he's trying to make an argument that it needs to go.

Nuke it from orbit and let's work on actually fixing the job instead of making it awful to play.

**Comment by necromanceranne** [comment]
If you got murdered by a pacifist botanist, who can't throw anything or shoot a syringe and therefore can't use most of botany's weaponization except slip memes, you almost assuredly wouldn't have survived even if he wasn't doping earthsblood.

I get its not a popular change and probably my suggestion was something better implemented after some more core QoL and mechanical changes to botany itself so they're not such chem pigs, but calling it an ided is missing the point. It'd be better to argue it's an ided because a botanist, in general, killed him and this is in an effort to curb lategame botany. But even then a botanist planning to kill people with plants doesn't have too much issue with or without golden trays.

**Comment by FancyCyclops** [comment]
> 
> 
> > Don't bother talking.
> 
> Hm? Are you saying that no one here can be reasoned with, or do you just not like long essays on why I think that certain changes would be bad ideas to make?

Well, what you say won't matter in the end. It's gonna get merged (or not) regardless of what you say.

**Comment by TheVekter** [comment]
> If you got murdered by a pacifist botanist, who can't throw anything or shoot a syringe and therefore can't use most of botany's weaponization except slip memes, you almost assuredly wouldn't have survived even if he wasn't doping earthsblood.
> 
> I get its not a popular change and probably my suggestion was something better implemented after some more core QoL and mechanical changes to botany itself so they're not such chem pigs, but calling it an ided is missing the point. It'd be better to argue it's an ided because a botanist, in general, killed him and this is in an effort to curb lategame botany. But even then a botanist planning to kill people with plants doesn't have too much issue with or without golden trays.

I don't so much care about his motivation as I do care about how bad these changes are. It takes a job that's already fairly boring and makes it worse. There's a reason Botany never interacted with the cook to the point of it being a meme - why would someone bust their ass making veggies or fruit when they can do funny weed meme? At least this way they actually make something other than weed and glowshrooms.

**Comment by actioninja** [comment]
Botany is definitely a hotspot of ridiculous bullshit, but making the game more annoying to play without actually removing any of the ridiculous bullshit isn't the solution. And that's all this is really doing. Earthsblood isn't a significant problem spot of botany even if it doesn't really fit into cobbychem.

**Comment by FancyCyclops** [comment]
> ...making the game more annoying to play without actually removing any of the ridiculous bullshit isn't the solution.

Advice you should have listened to.

**Comment by actioninja** [comment]
In conclusion
image

**Comment by ATH1909** [comment]
Let this be known as the day that we were reassured that the maintainers do indeed give a damn about botany. Thanks, actioninja.

**Comment by wesoda25** [comment]
this was a bad pr because i like the way the trays glow
